stripper/ze_l0v0l_v1_4_csgo2.cfg

```diff
@@ -28,4 +28,59 @@ modify:
 	{
 		"item_assaultsuit" "1"
 	}
-}
\ No newline at end of file
+}
+
+;fix level 4 nuke from killing every single player after the Nuke King boss is dead and you get teleported
+modify:
+{
+	match:
+	{
+		"targetname" "lvl4_win"
+	}
+	delete:
+	{
+		"OnStartTouch" "Map_BrushRunScriptCodeKillAll();61"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorAddOutputhealth 99999999990-1"
+		"OnStartTouch" "MapBrush,RunScriptCode,KillAllT();11"
+	}
+}
+
+	add:
+	{
+		"model" "*505"
+		"targetname" "lvl4_winfix"
+		"StartDisabled" "1"
+		"spawnflags" "1"
+		"origin" "-4487 5370 7350.65"
+		"filtername" "filter_zombie"
+		"classname" "trigger_multiple"
+		"OnStartTouch" "!activatorAddOutputhealth 99999999990-1"
+	}	
+modify:
+	{
+	match:
+	{
+		"targetname" "endrelay4"
+		"classname" "logic_relay"
+	}
+	insert:
+	{
+		"OnTrigger" "lvl4_winfixEnable01"
+	}
+}
```

Comment:
This shouldn't be necessary, since you're already setting everyone to 9999999999 in lvl4_win.